charts/kyverno-policies/templates/_helpers.tpl

```diff
@@ -0,0 +1,58 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "kyverno-policy.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "kyverno-policy.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/* Helm required labels */}}
+{{- define "kyverno-policy.labels" -}}
+app.kubernetes.io/component: kyverno
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/name: {{ template "kyverno-policy.name" . }}
+app.kubernetes.io/part-of: {{ template "kyverno-policy.name" . }}
+app.kubernetes.io/version: "{{ .Chart.Version }}"
+helm.sh/chart: {{ include "kyverno-policy.chart" . }}
+{{- end -}}
+
+{{/* Set if a baseline policy is managed */}}
```

Comment:
What is the value/function of this?

Reply:
We are defining a condition here to enable the policy based on the policy type. We have just defined one(baseline) policy type in values.yaml . Hence, this condition is redundant as of now as all the policies in the folder get enforced.